desc/plotting.py

```diff
@@ -3843,3 +3844,106 @@ def plot_logo(save_path=None, **kwargs):
+
+    Parameters
+    ----------
+    eq : object
+        Equilibrium object containing magnetic field information
+    rhos : array_like or float, optional
+        Flux surface radii. If float, plots single surface.
+        Default: np.linspace(0.1, 1, 10)
+    alphas : array_like, optional
+        Fieldline label values (toroidal angle).
+        Default: np.linspace(0, 2π, 32, endpoint=True)
+    num_pitch : int, optional
+        Number of pitch angle values for bounce integral calculation.
+        Default: 16
+
+    Returns
+    -------
+    fig : matplotlib.figure.Figure
+        The figure object containing the plot
+    ax : matplotlib.axes.Axes
+        The axes object for further customization
+
+    """
+    if rhos is None:
+        rhos = np.array([0.5])
+    elif isinstance(rhos, float):
+        rhos = np.array([rhos])
+
+    print("Calculating γ_c(α, λ)..")
+    if alphas is None:
+        alphas = np.linspace(0, 2 * np.pi, 32, endpoint=True)
+
+    if num_pitch is None:
+        num_pitch = 16
+
+    # TODO(#1352)
+    X, Y = 32, 64
+
+    from desc.integrals.bounce_integral import Bounce2D
+
+    # Compute bounce integral
+    theta = Bounce2D.compute_theta(eq, X, Y, rhos)
+    grid = LinearGrid(rho=rhos, M=eq.M_grid, N=eq.N_grid, NFP=eq.NFP, sym=False)
+    data0 = eq.compute(
+        "gamma_c",
+        grid=grid,
+        theta=theta,
+        Y_B=64,
+        num_transit=1,
+        num_quad=24,
+        num_pitch=num_pitch,
+        alpha=alphas,
+    )
+    data_full = grid.compress(data0["gamma_c"])
+    print("Plotting ...")
+
+    # Extract pitch angle range
+    minB = data0["min_tz |B|"][0]
+    maxB = data0["max_tz |B|"][0]
+    inv_pitch = np.linspace(minB, maxB, num_pitch)
+
+    # Create figure and prepare colormap
+    fig = plt.figure(figsize=(6, 5))
+    cmap = plt.get_cmap("plasma").copy()
+    cmap.set_under("white")  # Make values below vmin display as white
+
+    # Plot γ_c as function of α and 1/λ
+    extent = [inv_pitch.min(), inv_pitch.max(), alphas.min(), alphas.max()]
+    plt.imshow(
+        data_full[0],
+        origin="lower",  # α increases upward
+        extent=extent,
+        aspect="auto",  # allows aspect ratio to adjust
+        cmap=cmap,
+        interpolation="nearest",  # no smoothing
+    )
+
+    cbar = plt.colorbar()
+    cbar.ax.tick_params(labelsize=22)
+
+    # Format scientific notation
+    import matplotlib.ticker as ticker
+
+    cbar.ax.yaxis.set_major_formatter(ticker.ScalarFormatter(useMathText=True))
+    cbar.ax.yaxis.get_major_formatter().set_powerlimits((0, 0))
+    cbar.ax.yaxis.set_major_locator(ticker.MaxNLocator(6))
+
+    ax = plt.gca()
+    y_ticks = [0, np.pi / 2, np.pi, 3 * np.pi / 2, 2 * np.pi]
+    y_labels = ["0", r"$\pi/2$", r"$\pi$", r"$3\pi/2$", r"$2\pi$"]
+    ax.set_yticks(y_ticks)
+    ax.set_yticklabels(y_labels, fontsize=22)
+
+    # Add labels
+    plt.xlabel(r"$1/\lambda$", fontsize=24)
```

Comment:
I think this should have units of 1/T or something?

Reply:
Yeah. Done.